codalab/client/json_api_client.py

```diff
@@ -0,0 +1,299 @@
+from collections import defaultdict
+import urllib2
+
+from codalab.client.bundle_client import BundleClient
+from codalab.client.rest_client import RestClient
+from codalab.common import http_error_to_exception
+
+
+# TODO(sckoo): deal with more error cases
```

Comment:
Please do this part properly. Exception handling is important. See for example my implementation of wrap_exception.